src/config/HYPRE_config.h.in

```diff
@@ -29,6 +26,9 @@
 /* Define to 1 if you have the <stdint.h> header file. */
 #undef HAVE_STDINT_H
 
+/* Define to 1 if you have the <stdio.h> header file. */
+#undef HAVE_STDIO_H
```

Comment:
What's the reason for adding HAVE_STDIO_H and removing HAVE_MEMORY_H above? I'm not familiar with what these are actually doing anyway...

Reply:
This happened automatically somehow as a result of running the new 2.71 version of autoconf.  Not sure why.

Reply:
Oh, I thought all the files with .in were inputs that we set by hand as opposed to auto-generated files. Good to know.

Reply:
Generally they are, but sometimes autoconf will want to update these `.in` files as well.  It's a really confusing system.

Reply:
Note that I also ran an "update" script (forgot the name, but it was recommended by autoconf when I was doing all this).  Some things were changed as a result of that script.  All I know is I did NOT manually add HAVE_STDIO_H myself. :)

Reply:
Yep, it also got me. At the top it says `/* config/HYPRE_config.h.in. Generated from configure.in by autoheader. */`

Reply:
> forgot the name, but it was recommended by autoconf when I was doing all this

Probably `autoupdate`?